### ISSUE TITLE
Longer MOTD for server networks

### ISSUE THREAD

**Issue by seanpilkington** [Minehut/Meta#843]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**

In-Game

**Describe the feature**

Allow for longer MOTD's on server networks.

**How the feature is useful**

The current length of MOTD's doesn't allow you to show off your server enough and make it appealing.
Especially with the addition of server networks, there needs to be room to list all of your gamemodes and include any other info for players hovering over the server.

As you can see from the screenshot below, WildHub have quite a small MOTD and still can't fit it all on.
![image](https://user-images.githubusercontent.com/87049329/139842897-57c06a93-16b8-44ed-91ac-6a874743792a.png)
 

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Tringlle** [comment]
There can technically be 2 lines, I dont know why they arent supporting that as of now. The serverlist there are 2 lines however.

**Comment by seanpilkington** [comment]
> There can technically be 2 lines, I dont know why they arent supporting that as of now. The serverlist there are 2 lines however.

They do support multiple lines but theres still and over all character limit thats quite small.
I think it's only 64 characters and even then it gets cut off earlier for some reason.

**Comment by ReportCardsMC** [comment]
Yeah, the MOTD length is something that has been annoying for a long while now, if they increase it for networks, might as well increase it for all servers. It's been a thing for a while even before networks. Or at least have it so color codes don't count towards the MOTD like even Minehut has a longer MOTD than 64 characters, why limit us.

**Comment by CoreyShupe** [comment]
Agreed, idk why it’s limited like that.
